docs/analyzer/user_guide.md

```diff
@@ -348,13 +348,13 @@ checker configuration:
   Note that compiler errors and warnings are captured by CodeChecker only if it
   was emitted by clang-tidy.
 
-  Profiles
+  Checker labels
   ------------------------------------------------
-  In CodeCheckers there is a manual grouping of checkers. These groups are
-  called profiles. The collection of profiles is found in
-  config/checker_profile_map.json file. The goal of these profile is that you
-  can enable or disable checkers by these profiles. See the output of
-  "CodeChecker checkers --profile list" command.
+  In CodeChecker there is a manual grouping of checkers. These groups are
+  determined by labels. The collection of labels is found in
+  config/checker_labels.json file. The goal of these labels is that you can
```

Comment:
This file is removed, so the documentation need to be updated here. Please search this file name, I think it occures more places in the code.